dotnet/src/IntegrationTests/Connectors/OpenAI/OpenAICompletionTests.cs

```diff
@@ -351,6 +352,103 @@ public async Task AzureOpenAIDefaultValueTestAsync()
         Assert.Contains("Bob", actual.Result, StringComparison.OrdinalIgnoreCase);
     }
 
+    [Fact]
+    public async Task MultipleServiceInlineServiceIdTestAsync()
```

Comment:
While it's good to have more integration tests, I'm wondering if scenario in this PR can be verified with unit tests. Because these tests will make a call to Azure OpenAI endpoint every time, but they verify the logic with service registration only. With proper mocking it should be possible to verify which service is used during function invocation without actual call to Azure OpenAI. This should save some resources.

Reply:
I'll replace this test with unit tests